broker/tasks/alb.py

```diff
@@ -162,6 +162,10 @@ def add_certificate_to_alb(operation_id, **kwargs):
     service_instance = operation.service_instance
     certificate = service_instance.new_certificate
 
+    if not certificate:
```

Comment:
I'm not sure if this is a good fix at all, but it is attempting to address this error which was observed in the logs

```
File "/home/vcap/app/broker/tasks/alb.py", line 172, in add_certificate_to_alb,     Certificates=[{"CertificateArn": certificate.iam_server_certificate_arn}],,                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^, AttributeError: 'NoneType' object has no attribute 'iam_server_certificate_arn'
```

Reply:
I think this will silence that error, but cause another one down the line when we try to do something with e.g, the cert arn

Reply:
OK. yeah, I'm not sure what else to do, because it's hard to trace the original cause

Reply:
I'm not seeing these errors right now, so I'll remove this possibly unhelpful defensive programming